sympy/stats/crv_types.py

```diff
@@ -3167,14 +3167,14 @@ def Normal(name, mean, std):
     The density of the Normal distribution is given by
 
     .. math::
-        f(x) := \frac{1}{\sigma\sqrt{2\pi}} e^{ -\frac{(x-\mu)^2}{2\sigma^2} }
+        f(x) := \frac{1}{\std\sqrt{2\pi}} e^{ -\frac{(x-\mean)^2}{2\std^2} }
 
     Parameters
     ==========
 
-    mu : Real number or a list representing the mean or the mean vector
-    sigma : Real number or a positive definite square matrix,
-         :math:`\sigma^2 > 0`, the variance
+    mean : Real number or a list representing the mean or the mean vector
+    std : Real number or a positive definite square matrix,
+         :math:`\std^2 > 0`, the variance
```

Comment:
I don't think that any other math except this should be changed though.
`\std, \mean` may not be a valid LaTeX functions

Reply:
Thanks for the review, Lee. I realized that \std and \mean are not valid Latex functions.